config/settings.yml

```diff
@@ -136,6 +136,9 @@ overpass_credentials: false
 graphhopper_url: "https://graphhopper.com/api/1/route"
 fossgis_osrm_url: "https://routing.openstreetmap.de/"
 fossgis_valhalla_url: "https://valhalla1.openstreetmap.de/route"
+# Endpoints for Wikimedia integration
+wikidata_url: "https://www.wikidata.org/w/api.php"
```

Comment:
A better name is `wikidata_api_url` because what if you want to generate a link to the wikidata web page, it won't have `w/api.php` in it. Or maybe add `w/api.php` in `explainWikidataLink` when building an url.